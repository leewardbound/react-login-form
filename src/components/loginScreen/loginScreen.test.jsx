import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import LoginScreen from '.';

const renderer = new ShallowRenderer();

describe('LoginScreen', function () {
  it('should render correctly', function () {
    const component = renderer.render(
      <LoginScreen />
    );

    expect(component).toMatchSnapshot();
  });
});
