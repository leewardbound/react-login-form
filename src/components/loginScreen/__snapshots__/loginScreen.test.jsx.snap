// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreen should render correctly 1`] = `
<styled.div>
  <styled.div>
    <styled.div
      md={6}
      mdOffset={3}
      sm={8}
      smOffset={2}
    >
      <Connect(branch(LoginFormContainer))
        title="Hello again"
      >
        <Connect(withState(withState(withPropsOnChange(withHandlers(LoginForm))))) />
      </Connect(branch(LoginFormContainer))>
    </styled.div>
  </styled.div>
</styled.div>
`;
